dracut/30ignition/ignition-diskful-subsequent.target

```diff
@@ -0,0 +1,12 @@
+# This target is a combination of ignition-subsequent.target and
+# ignition-diskful.target - units here should only run when we have a
+# boot disk and *aren't* doing the first boot.
+[Unit]
+Description=Ignition Subsequent Boot Disk Setup
+Documentation=https://github.com/coreos/ignition
+ConditionPathExists=/etc/initrd-release
+Before=ignition-complete.target
```

Comment:
I think this should be `ignition-subsequent.target`, right?

Reply:
Yep, thanks!